index.html

```diff
@@ -28,18 +29,30 @@
 
         
         <nav>
-            <img class="nav" src="Images/menu-button.png" alt="">
+            <img class="nav" id="myBtn" src="Images/menu-button.png" alt="">
             <ul class="nav-link">
                 <li><a href="">Portfolio</a></li>
                 <li><a href="">About</a></li>
                 <li><a href="">Contact</a></li>
             </ul>
         </nav>
+        <!-- Modal content -->
+        <div id="myModal" class="modal">
+        <div class="modal-content">
+            <span class="close">&times;</span>
+            <ul class="modal-list">
```

Comment:
- [ ] Kindly  it will be better if you decrease the margin-top of the icon x in the mobile menu as you see it is different from the Figma Design

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/225032766-29fa785e-6dbd-409c-a07f-4d43c6144431.png)    |    ![image](https://user-images.githubusercontent.com/83360636/225032956-1a89b15e-6fcf-4efe-aef8-debe270f225b.png)  |